### ISSUE TITLE
Discuss: Screenshot for accessibility failures are confusing, but removing them may be confusing too.

### ISSUE THREAD

**Issue by plocket** [plocket/docassemble-cucumber#748]
A problem with accessibility errors is that, even when they're the only error, the test still takes a screenshot of the final page of the test, even though that page isn't relevant. Maybe we should avoid taking a screenshot at the end if the only failures are accessibility failures.

Additionally, should we take a screenshot of the page/field where the accessibility issue arose? [Conclusion: We should take the a11y error pic just after adding the a11y failure info to the report.]

**Comment by BryceStevenWilley** [comment]
If we start saving the DOM as HTML as well as screenshoting (see #763), that would be very useful for accessibility issues.

**Comment by plocket** [comment]
That's true, but the screenshot itself is still not relevant and may be confusing/misleading to the dev. We'd still need to make sure to take screenshots if there were other errors, of course.

**Comment by BryceStevenWilley** [comment]
> the screenshot itself is still not relevant and may be confusing/misleading to the dev

Should have clarified; I agree that the current situation of taking a screenshot of the final page isn't useful. However, a screenshot of the page that the accessibility error is on would still be useful. Sure, it doesn't fit in with our current UX of encouraging people to look through the screenshots to see the errors, but the screenshot is still useful to have when looking for certain accessibility issues.

**Comment by plocket** [comment]
I think we're actually on the same page and I mistook what you were referring to in your comment. All sounds good 👍

**Comment by plocket** [comment]
We should take the a11y error pic just after adding the a11y failure info to the report.
